ci-operator/step-registry/openshift/microshift/infra/iso-boot/openshift-microshift-infra-iso-boot-ref.yaml

```diff
@@ -10,9 +10,11 @@ ref:
   env:
   - name: HOME
     default: /tmp/secret
+  - name: SCENARIO_TYPE
+    default: presubmits
```

Comment:
If I were to add a **periodic** job and forget to define the `SCENARIO_TYPE`, what would happen? Does the CI return a false positive signal?

Reply:
Ended up removing the default as per @copejon suggestion. This makes the whole step fail if its not defined, ensuring there are no false positives/unexpected tests.